scripts/CheckSecurityConfigs.s.sol

```diff
@@ -90,9 +90,9 @@ contract CheckSecurityConfigs is Script {
         // Only testnet chains may be added as an exception here.
         return keccak256(
             abi.encodePacked(
-                sliceString(addressesJsonPath, bytes(addressesJsonPath).length - 15, bytes(addressesJsonPath).length)
+                sliceString(addressesJsonPath, bytes(addressesJsonPath).length - 18, bytes(addressesJsonPath).length)
             )
-        ) == keccak256(abi.encodePacked("sepolia/op.json"));
```

Comment:
Really keen to replace this code (see https://github.com/ethereum-optimism/superchain-registry/pull/144).

Reply:
Just pushed 3e61dba to clean this up, feel free to revert the commit if you prefer